### PR TITLE
[DAT-16135] add new scope flag for modifyChangeSet flow

### PR DESCRIPTION
**Pull request by KushnirykOleh**
## Impact

- [x] Bug fix (non-breaking change which fixes expected existing functionality)
 

## Description
For some parsers, (like mongosh parsers) we need to know when modifyChangeSet flow is happening, as we can't run mongo instructions without native executor so `runWith` is required. But in modifyChangeSet flow `runWith` is added to the changeset later on, after parsing. 
Adding modifyChangeSets flag to the Scope allows parsers to read it and skip runWith validation knowing it's inner changeset and it gets 'runWith'  attribute later.

### CODE REVIEW

**Review by wwillard7800 — APPROVED** [review]
Looks good.

File: liquibase-standard/src/main/java/liquibase/changelog/DatabaseChangeLog.java [hunk 1]
```diff
@@ -924,11 +924,12 @@ public boolean include(String fileName,
         if (isRelativePath) {
             try {
                 fileName = resourceAccessor.get(this.getPhysicalFilePath()).resolveSibling(fileName).getPath();
-                fileName = normalizePath(Paths.get(fileName).toString());
             } catch (IOException e) {
                 throw new UnexpectedLiquibaseException(e);
             }
         }
+        final String normalizedFilePath = normalizePath(Paths.get(fileName).toString());
```
Comment (StevenMassaro) [inline liquibase-standard/src/main/java/liquibase/changelog/DatabaseChangeLog.java:931]: Is there a reason you moved this from its original location?

Comment (KushnirykOleh) [reply liquibase-standard/src/main/java/liquibase/changelog/DatabaseChangeLog.java:931]: had to make it final to pass it to the runner

Comment (StevenMassaro) [reply liquibase-standard/src/main/java/liquibase/changelog/DatabaseChangeLog.java:931]: As long as moving it outside of the `if(isrelativePath)` block is okay, then I am fine with this.

**Review by StevenMassaro — APPROVED** [review]

**Review by filipelautert — APPROVED** [review]

**Pull request merged**